src/Protocol/Models/InsertReplaceEdit.cs

```diff
@@ -0,0 +1,33 @@
+using System.Diagnostics;
+
+namespace OmniSharp.Extensions.LanguageServer.Protocol.Models
+{
+    /// <summary>
+    /// A special text edit to provide an insert and a replace operation.
+    ///
+    /// @since 3.16.0 - proposed state
+    /// </summary>
+    [DebuggerDisplay("{" + nameof(DebuggerDisplay) + ",nq}")]
+    public class InsertReplaceEdit
```

Comment:
This will be added onto the `CompletionItem` but this is a breaking change so this has not been put into place yet.